sys/net/gnrc/network_layer/sixlowpan/nd/gnrc_sixlowpan_nd.c

```diff
@@ -372,6 +372,7 @@ uint8_t gnrc_sixlowpan_nd_opt_ar_handle(kernel_pid_t iface, ipv6_hdr_t *ipv6,
 
 bool gnrc_sixlowpan_nd_opt_6ctx_handle(uint8_t icmpv6_type, sixlowpan_nd_opt_6ctx_t *ctx_opt)
 {
+#ifdef MODULE_GNRC_SIXLOWPAN_CTX
```

Comment:
I would just exclude the whole function and replace the function definition with a macro evaluating to `true` in case `MODULE_GNRC_SIXLOWPAN_CTX` is not defined. Similarly how I did it with the router functionality of NDP.